C++/include/algorithm/string/edit_distance.hpp

```diff
@@ -0,0 +1,74 @@
+/*
+Edit Distance Algorithm
+---------------------------------
+This algorithm finds the minimum number of edits which are required to convert string1 to string2.
+Edit here means either insert,remove or replace. The algorithm implemented here uses dynamic programming.
+
+Time Complexity
+---------------------------------
+O(m*n)  
+where-
+m is length of string1
+n is length of string2
+
+Space Complexity
+---------------------------------
+O(m*n)
+where-
+m is length of string1
+n is length of string2
+
+*/
+
+
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+using namespace std;
```

Comment:
Avoid `using namespace std`. Instead (for example): `using std::vector`.

Reply:
Like @alxmjo said, this should be avoided in header files as this can lead to namespace pollution when this code is included elsewhere.